lesson04/assignment/index.html

```diff
@@ -5,11 +5,36 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="stylesheet" href="css/style.css">
+
   <title>Document</title>
 </head>
 <body>
-  <main class="main">
 
-  </main>
+
+<nav>
+    <div class="hamburger">
+      <div class="line"></div>
+      <div class="line"></div>
+      <div class="line"></div>
+    </div>
+    <ul class="nav-links">
+
+      <li><a href="#">Home</a></li>
+      <li><a href="#">Music</a></li>
+      <li><a href="#">Contact</a></li>
+    </ul>
+</nav>
+
+<div class="small">
+  <h1>Music</h1>
+</div>
+
+
+
+
+
+
+
+  <script src="app.js"></script>
 </body>
```

Comment:
I'm not seeing any of the jpg or gif images used in this file. They shouldn't be included in the repo if they're not used. We only want necessary files in our repo.